note Lambda names follow DD tagging format (#10073)
DataDog/documentation: content/en/serverless/serverless_tagging/_index.md

To tag your serverless application with `env`, `service` and `version`, see the [unified service tagging documentation][1].

> Also note that Lambda function names should adhere to [Datadog's tagging convention][4]. This allows us to tie all of your function's traces, logs and metrics together seamlessly.

Reviewer: ```suggestion
**Note**: Lambda function names should adhere to [Datadog's tagging convention][4]. This ties all of your function's traces, logs and metrics together seamlessly.
```
